Client/mods/deathmatch/logic/CResourceFileDownloadManager.cpp

```diff
@@ -137,9 +148,16 @@ void CResourceFileDownloadManager::DoPulse()
     }
 
     // Update progress box
-    GetTransferBox()->SetInfo(uiDownloadSizeTotal);
+    GetTransferBox()->SetDownloadProgress(uiDownloadSizeTotal);
     GetTransferBox()->DoPulse();
 
+    // Call Lua event 'onClientTransferBoxProgressChange'
+    CLuaArguments arguments;
+    arguments.PushNumber(uiDownloadSizeTotal);
```

Comment:
Might be worth renaming `uiDownloadSizeTotal` to `uiDownloadedSizeTotal` to make this a bit more clear